packages/react/src/themes/teams/components/Popup/popupStyles.ts

```diff
@@ -8,6 +8,8 @@ const popupStyles: ComponentSlotStylesInput<PopupProps, PopupVariables> = {
   popup: ({ variables }): ICSSInJSStyle => ({
     zIndex: variables.zIndex,
     position: 'absolute',
+    color: variables.contentColor,
+    background: variables.contentBackgroundColor,
```

Comment:
Is there a reason why we moved the background style from the `PopupContent` to the `Popup`? I'd prefer to see them in it's own component styles, rather then in the popup, especially because they are not anyhow dependent on some props from the `Popup` component.

Reply:
yes - the reason is that we want to see the background and foreground colors being applied to the popup's content _irregardless of what is rendered as popup content_. Previously we have introduced proper styles only for the cases when `PopupContent` was rendered at the `content` slot - however, in case if user would render plain content there, there won't be any background/foreground styles applied at all. Thus, the following code

```jsx
<Popup ... content={<div>Hello from popup content!</div>} />
``` 

would result in the presentation that was mentioned in the PR's description:

![image](https://user-images.githubusercontent.com/9024564/55152477-c8d2c300-5150-11e9-90c9-5f10177863ba.png)

However, what client really wants here is proper basic styles applied even if plain content is provided to the slot:

![image](https://user-images.githubusercontent.com/9024564/55152549-e6079180-5150-11e9-975c-0d75ff17c8fa.png)

That's why the variables were moved - note that with that we are not limiting client in overriding styles for the content in case if necessary (so, if `content` provides its own styles for `background` and `color`, then those will be used)